src/QsCompiler/Transformations/Monomorphization.cs

```diff
@@ -359,9 +378,13 @@ private class ExpressionKindTransformation : ExpressionKindTransformation<Transf
                             .Where(kvp => kvp.Key.Item1.Equals(global.Item))
                             .ToImmutableDictionary(kvp => kvp.Key, kvp => kvp.Value);
 
-                        // Create a new identifier
-                        sym = SharedState.GetConcreteIdentifier(global, applicableParams);
-                        tArgs = QsNullable<ImmutableArray<ResolvedType>>.Null;
+                        // We want to skip over intrinsic callables. They will not be monomorphized.
```

Comment:
It's probably worth pointing this out in the doc comment of the transformation.

Reply:
I have added it to the doc comment.